### ISSUE TITLE
ScyllaDB/Cassandra - unable to apply migrations

### ISSUE THREAD

**Issue by petyots** [flyway/flyway#3988]
###### Which version and edition of Flyway are you using?
```flyway version
Flyway OSS Edition 11.0.0 by Redgate

See release notes here: https://rd.gt/416ObMi
Plugin Name     | Version
--------------- | ---------
TiDB            | 10.16.0
OceanBase       | 10.16.0
DB2 for z/OS    | 10.16.3
YugabyteDB      | 10.16.0
Apache Ignite   | 10.16.0
ClickHouse      | 10.16.0
```

###### If this is not the latest version, can you reproduce the issue with the latest one as well? (Many bugs are fixed in newer releases and upgrading will often resolve the issue)
Yes, my installation of flyway is the latest publicly available

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin)
I am using the CLI client on M2 Max(aarm64)

###### Which database are you using? (Type & version)
[cqlsh 6.0.23.dev3+g86a280a | Scylla 6.1.2-0.20240915.b60f9ef4c223 | CQL spec 3.3.1 | Native protocol v4]

###### Which operating system are you using?
macOS Sequoia 15.1.1 (24B91)

###### What did you do? (Please include the content causing the issue, any relevant configuration settings, the SQL statement(s) that failed (if any), and the command you ran)
I tried to run migrations against my ScyllaDB using `flyway migrate`

`config.toml`:
```toml
[flyway]
locations = ["filesystem:./src/API/Shared/Persistence/Migrations"]
sqlMigrationSuffixes = ["*.cql"]
defaultSchema = "project"

[environments.default]
url = "jdbc:cassandra://localhost:9042/project"
user = ""
password = ""
```

###### What did you expect to see?
The migrations to be applied successfully

###### What did you see instead?

```
flyway migrate
Flyway OSS Edition 11.0.0 by Redgate

See release notes here: https://rd.gt/416ObMi
Exception in thread "main" java.lang.NoClassDefFoundError: io/netty/util/concurrent/FastThreadLocalThread
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1026)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
	at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:862)
	at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:760)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:681)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:639)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:528)
	at com.datastax.oss.driver.internal.core.util.concurrent.BlockingOperation.checkNotDriverThread(BlockingOperation.java:44)
	at com.datastax.oss.driver.api.core.session.SessionBuilder.build(SessionBuilder.java:834)
	at com.ing.data.cassandra.jdbc.SessionHolder.createSession(SessionHolder.java:367)
	at com.ing.data.cassandra.jdbc.SessionHolder.<init>(SessionHolder.java:134)
	at com.ing.data.cassandra.jdbc.CassandraDriver$1.load(CassandraDriver.java:63)
	at com.ing.data.cassandra.jdbc.CassandraDriver$1.load(CassandraDriver.java:60)
	at com.github.benmanes.caffeine.cache.LocalLoadingCache.lambda$newMappingFunction$2(LocalLoadingCache.java:145)
	at com.github.benmanes.caffeine.cache.UnboundedLocalCache.lambda$computeIfAbsent$2(UnboundedLocalCache.java:239)
	at java.base/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1713)
	at com.github.benmanes.caffeine.cache.UnboundedLocalCache.computeIfAbsent(UnboundedLocalCache.java:235)
	at com.github.benmanes.caffeine.cache.LocalCache.computeIfAbsent(LocalCache.java:108)
	at com.github.benmanes.caffeine.cache.LocalLoadingCache.get(LocalLoadingCache.java:56)
	at com.ing.data.cassandra.jdbc.CassandraDriver.connect(CassandraDriver.java:90)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:275)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:240)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:59)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:76)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:137)
	at org.flywaydb.core.Flyway.migrate(Flyway.java:177)
	at org.flywaydb.commandline.Main.executeOperation(Main.java:301)
	at org.flywaydb.commandline.Main.executeFlyway(Main.java:207)
	at org.flywaydb.commandline.Main.main(Main.java:129)
Caused by: java.lang.ClassNotFoundException: io.netty.util.concurrent.FastThreadLocalThread
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:528)
	... 31 more
```
	
	
###### Additional debug info
```bash
flyway info -X
DEBUG: Java Version: 23.0.1
DEBUG: Default configuration:
DEBUG: environments.default.flyway -> {}
environments.default.jdbcProperties -> {}
environments.default.schemas -> []
flyway.baselineDescription -> << Flyway Baseline >>
flyway.baselineOnMigrate -> false
flyway.batch -> false
flyway.callbacks -> []
flyway.cleanDisabled -> true
flyway.cleanOnValidationError -> false
flyway.communityDBSupportEnabled -> true
flyway.createSchemas -> true
flyway.detectEncoding -> false
flyway.encoding -> UTF-8
flyway.environment -> default
flyway.errorOverrides -> []
flyway.executeInTransaction -> true
flyway.failOnMissingLocations -> false
flyway.failOnMissingTarget -> false
flyway.group -> false
flyway.ignoreMigrationPatterns -> [*:future]
flyway.jarDirs -> []
flyway.kerberosConfigFile ->
flyway.locations -> [db/migration]
flyway.lockRetryCount -> 50
flyway.loggers -> [auto]
flyway.migrationResolvers -> []
flyway.mixed -> false
flyway.outOfOrder -> false
flyway.outputProgress -> false
flyway.outputQueryResults -> true
flyway.placeholderPrefix -> ${
flyway.placeholderReplacement -> true
flyway.placeholderSeparator -> :
flyway.placeholderSuffix -> }
flyway.placeholders -> {}
flyway.provisionMode -> provision
flyway.repeatableSqlMigrationPrefix -> R
flyway.reportEnabled -> false
flyway.reportFilename -> report
flyway.scriptPlaceholderPrefix -> FP__
flyway.scriptPlaceholderSuffix -> __
flyway.skipDefaultCallbacks -> false
flyway.skipDefaultResolvers -> false
flyway.skipExecutingMigrations -> false
flyway.sqlMigrationPrefix -> V
flyway.sqlMigrationSeparator -> __
flyway.sqlMigrationSuffixes -> [.sql]
flyway.stream -> false
flyway.table -> flyway_schema_history
flyway.target -> latest
flyway.validateMigrationNaming -> false
flyway.validateOnMigrate -> true

DEBUG: Loading config file: /Users/me/Developer/Projects/Project/ProjectApi/flyway.toml
DEBUG: environments.default.flyway -> {}
environments.default.jdbcProperties -> {}
environments.default.password ->
environments.default.schemas -> []
environments.default.url -> jdbc:cassandra://localhost:9042/project
environments.default.user ->
flyway.defaultSchema -> project
flyway.locations -> [filesystem:./src/API/Shared/Persistence/Migrations]
flyway.sqlMigrationSuffixes -> [*.cql]

DEBUG: Skipping empty environment variables
DEBUG: No flyway namespace variables found in command line
DEBUG: Using configuration:
DEBUG: environments.default.flyway -> {}
environments.default.jdbcProperties -> {}
environments.default.password ->
environments.default.schemas -> []
environments.default.url -> jdbc:cassandra://localhost:9042/project
environments.default.user ->
flyway.baselineDescription -> << Flyway Baseline >>
flyway.baselineOnMigrate -> false
flyway.batch -> false
flyway.callbacks -> []
flyway.cleanDisabled -> true
flyway.cleanOnValidationError -> false
flyway.communityDBSupportEnabled -> true
flyway.createSchemas -> true
flyway.defaultSchema -> project
flyway.detectEncoding -> false
flyway.encoding -> UTF-8
flyway.environment -> default
flyway.errorOverrides -> []
flyway.executeInTransaction -> true
flyway.failOnMissingLocations -> false
flyway.failOnMissingTarget -> false
flyway.group -> false
flyway.ignoreMigrationPatterns -> [*:future]
flyway.jarDirs -> []
flyway.kerberosConfigFile ->
flyway.locations -> [filesystem:./src/API/Shared/Persistence/Migrations]
flyway.lockRetryCount -> 50
flyway.loggers -> [auto]
flyway.migrationResolvers -> []
flyway.mixed -> false
flyway.outOfOrder -> false
flyway.outputProgress -> false
flyway.outputQueryResults -> true
flyway.placeholderPrefix -> ${
flyway.placeholderReplacement -> true
flyway.placeholderSeparator -> :
flyway.placeholderSuffix -> }
flyway.placeholders -> {}
flyway.provisionMode -> provision
flyway.repeatableSqlMigrationPrefix -> R
flyway.reportEnabled -> false
flyway.reportFilename -> report
flyway.scriptPlaceholderPrefix -> FP__
flyway.scriptPlaceholderSuffix -> __
flyway.skipDefaultCallbacks -> false
flyway.skipDefaultResolvers -> false
flyway.skipExecutingMigrations -> false
flyway.sqlMigrationPrefix -> V
flyway.sqlMigrationSeparator -> __
flyway.sqlMigrationSuffixes -> [*.cql]
flyway.stream -> false
flyway.table -> flyway_schema_history
flyway.target -> latest
flyway.validateMigrationNaming -> false
flyway.validateOnMigrate -> true

DEBUG: Scanning for classpath resources at 'classpath:db/callback' ...
DEBUG: Determining location urls for classpath:db/callback using ClassLoader java.net.URLClassLoader@15de0b3c ...
DEBUG: Unable to resolve location classpath:db/callback.
Flyway OSS Edition 11.0.0 by Redgate

See release notes here: https://rd.gt/416ObMi
DEBUG: Scanning for filesystem resources at './src/API/Shared/Persistence/Migrations'
DEBUG: Scanning for resources in path: ./src/API/Shared/Persistence/Migrations (./src/API/Shared/Persistence/Migrations)
DEBUG: Found filesystem resource: ./src/API/Shared/Persistence/Migrations/V1__create_users.cql
Exception in thread "main" java.lang.NoClassDefFoundError: io/netty/util/concurrent/FastThreadLocalThread
at java.base/java.lang.ClassLoader.defineClass1(Native Method)
at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1026)
at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:862)
at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:760)
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:681)
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:639)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:528)
at com.datastax.oss.driver.internal.core.util.concurrent.BlockingOperation.checkNotDriverThread(BlockingOperation.java:44)
at com.datastax.oss.driver.api.core.session.SessionBuilder.build(SessionBuilder.java:834)
at com.ing.data.cassandra.jdbc.SessionHolder.createSession(SessionHolder.java:367)
at com.ing.data.cassandra.jdbc.SessionHolder.<init>(SessionHolder.java:134)
at com.ing.data.cassandra.jdbc.CassandraDriver$1.load(CassandraDriver.java:63)
at com.ing.data.cassandra.jdbc.CassandraDriver$1.load(CassandraDriver.java:60)
at com.github.benmanes.caffeine.cache.LocalLoadingCache.lambda$newMappingFunction$2(LocalLoadingCache.java:145)
at com.github.benmanes.caffeine.cache.UnboundedLocalCache.lambda$computeIfAbsent$2(UnboundedLocalCache.java:239)
at java.base/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1713)
at com.github.benmanes.caffeine.cache.UnboundedLocalCache.computeIfAbsent(UnboundedLocalCache.java:235)
at com.github.benmanes.caffeine.cache.LocalCache.computeIfAbsent(LocalCache.java:108)
at com.github.benmanes.caffeine.cache.LocalLoadingCache.get(LocalLoadingCache.java:56)
at com.ing.data.cassandra.jdbc.CassandraDriver.connect(CassandraDriver.java:90)
at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:275)
at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:240)
at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:59)
at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:76)
at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:137)
at org.flywaydb.core.Flyway.info(Flyway.java:263)
at org.flywaydb.commandline.Main.executeOperation(Main.java:328)
at org.flywaydb.commandline.Main.executeFlyway(Main.java:207)
at org.flywaydb.commandline.Main.main(Main.java:129)
Caused by: java.lang.ClassNotFoundException: io.netty.util.concurrent.FastThreadLocalThread
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:528)
... 31 more
```

**Comment by JasonLuo-Redgate** [comment]
Hi @petyots ,
Thanks for reporting this issue. The fix has been included in the latest `11.0.1` release. 
Could you try that build to check if the issue has been resolved?

**Comment by petyots** [comment]
Hi @JasonLuo-Redgate,

Thank you very much. It works!
